app/hall/fighters.py

```diff
@@ -0,0 +1,8 @@
+# from __future__ import annotations
+
+
+class Fighters(object):
+    def __init__(self, fighter: dict) -> None:
+        for key in fighter:
+            setattr(self, key, fighter[key])
```

Comment:
is it not clear logic, pls add arguments

Reply:
Pls, give me a hint, because in the last commit I had this problem required several times as many lines of code

Reply:
You need to set only specific key:
hp, protection, power

Reply:
@neagarr not fixed
add clear logic to init